lib/new_relic/agent/instrumentation/redis/instrumentation.rb

```diff
@@ -6,38 +6,58 @@ module NewRelic::Agent::Instrumentation
   module Redis
     PRODUCT_NAME = 'Redis'
     CONNECT = 'connect'
-    UNKNOWN = "unknown"
-    LOCALHOST = "localhost"
+    UNKNOWN = 'unknown'
+    LOCALHOST = 'localhost'
     MULTI_OPERATION = 'multi'
     PIPELINE_OPERATION = 'pipeline'
+    REDIS_5 = Gem::Version.new(::Redis::VERSION) >= Gem::Version.new('5.0.0') && !!defined?(::RedisClient)
```

Comment:
Given the `>=` nature of this indicator, I'd prefer a constant name that was more futureproof (for Redis 6) and specific in its description of what will happen if it is `true`. Possibly `USES_REDIS_CLIENT` or `HAS_REDIS_CLIENT` might work to signify that the separately namespaced redis client class.

Reply:
Great point. Resolved in: a8198510ec8eaae23e51cb5271ef7eb250dd28a7